kiva/fonttools/font.py

```diff
@@ -98,21 +193,35 @@ class Font(object):
     def __init__(self, face_name="", size=12, family=SWISS,
                  weight=WEIGHT_NORMAL, style=NORMAL, underline=0,
                  encoding=DEFAULT):
-        if (not isinstance(face_name, str)
-                or not isinstance(size, int)
-                or not isinstance(family, int)
-                or not isinstance(weight, int)
-                or not isinstance(style, int)
-                or not isinstance(underline, int)
-                or not isinstance(encoding, int)):
+        if not isinstance(face_name, str):
+            raise RuntimeError(
+                f"Expected face name to be a str, got {face_name!r}")
+        if not isinstance(size, int):
+            raise RuntimeError(
+                f"Excected size to be an int, got {size!r}")
+        if not isinstance(family, int):
+            raise RuntimeError(
+                f"Excected family to be an int, got {family!r}")
+        if not isinstance(weight, int):
+            raise RuntimeError(
+                f"Excected weight to be an int, got {weight!r}")
+        if not isinstance(style, int):
+            raise RuntimeError(
+                f"Excected style to be an int, got {style!r}")
+        if not isinstance(underline, int):
+            raise RuntimeError(
+                f"Excected underline to be a int, got {underline!r}")
+        if not isinstance(encoding, int):
```

Comment:
Mismatch between the checked type (`int`) and the type in the error message (`bool`) here. I'm not sure which to believe: what's `encoding` for?

Reply:
Probably should be an int, given that was what was being checked for - `bool` is likely a copy-paste error in this case